deployment/tokens/internal/ops/ops_evm.go

```diff
@@ -0,0 +1,91 @@
+package ops
+
+import (
+	"github.com/Masterminds/semver/v3"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+
+	cldf "github.com/smartcontractkit/chainlink-deployments-framework/deployment"
+	"github.com/smartcontractkit/chainlink-deployments-framework/operations"
+	link_token "github.com/smartcontractkit/chainlink-testing-framework/seth/contracts/bind/link"
```

Comment:
Should we be using Link token from the CTF package & not from chainlink-evm?

Reply:
Good catch! Auto imports fails